x/bet/genesis.go

```diff
@@ -9,9 +9,23 @@ import (
 // InitGenesis initializes the module's state from a provided genesis
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+
+	for _, elem := range genState.Uid2IdList {
+		k.SetBetID(ctx, elem)
```

Comment:
SetBet on line 28, in same file, calls SetBetID again. Need to discuss implications of this.